libphal/phal_pdbg.C

```diff
@@ -130,6 +132,76 @@ void deconfigureTgt(const ATTR_PHYS_BIN_PATH_Type &physBinPath,
 	}
 }
 
+void getLocationCode(struct pdbg_target *target,
+		     ATTR_LOCATION_CODE_Type &locationCode)
+{
+	// check target is valid.
+	if (target == nullptr) {
+		log(level::ERROR, "getLocationCode: Invalid pdbg Target");
+		throw pdbgError_t(exception::PDBG_TARGET_NOT_FOUND);
```

Comment:
JS: PDBG_TARGET_INVALID?

Reply:
it is is better to get new one. updated.